index.html

```diff
@@ -0,0 +1,255 @@
+      </div>
+      </div>
+    </section>
+    <section class="card_item">
+      <div class="snap">
+        <img src="background/snap3/Snapshoot Portfolio.png" alt="">
+      </div>
+        <h1 class="spa">Tonic</h1><br>
+        <ul class="list">
+          <li class="canopy"><b> CANOPY</b></li>
+          <li><img src="background/cano/Counter.png" alt=""></li>
+          <li>Back End Dev</li>
+          <li><img src="background/cano/Counter.png" alt=""></li>
+          <li>2015</li>
+        </ul><br>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="text">
+            <li class="lists">html</li>
+            <li class="lists">css</li>
+            <li class="lists">javascript</li>
+          </ul> <br>
+          <button id="bttn">see project</button>
+    </section>
+    <section class="card_item">
+      <div class="snap">
+        <img src="background/snap4/Snapshoot Portfolio.png" alt="">
+      </div>
+        <h1 class="spa"> Multi-Post Stories</h1>
+      <ul class="list">
+        <li class="canopy"><b> CANOPY</b></li>
+        <li><img src="background/cano/Counter.png" alt=""></li>
+        <li>Back End Dev</li>
+        <li><img src="background/cano/Counter.png" alt=""></li>
+        <li>2015</li>
+      </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="text">
+          <li class="lists">html</li>
+          <li class="lists">css</li>
+          <li class="lists">javascript</li>
+        </ul> <br>
+          <button id="bttn">see project</button>
+    </section>
+  </section>
+
+  <section class="section6">
+    <div>
+      <h2 class="about">About <br>Myself</h2><br>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p><br>
+      <a href="" class="lets">lets connect
+      </a>
+    </div>
+    <br>
+    <div>
+      <a href="">
+        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none">
+          <path
+            d="M19.7587 0.394423C19.4253 -0.00424693 18.8587 -0.103914 18.4253 0.195088L16.492 1.52399C15.592 0.560536 14.3253 0.0289756 12.9587 0.0289756C10.2587 0.0289756 8.02532 2.22166 8.02532 4.94591C8.02532 5.34458 8.05865 5.71002 8.15865 6.07547C5.82532 5.57713 4.42532 3.88279 3.89198 0.826316C3.82532 0.427646 3.52532 0.128643 3.12532 0.0289756C2.75865 -0.070692 2.35865 0.0954206 2.12532 0.394423C2.05865 0.460868 -3.44135 8.10205 4.52532 14.5804C3.75865 16.2083 1.45865 17.6037 0.558651 18.0688C0.125318 18.2681 -0.0746825 18.7332 0.0253175 19.1984C0.125318 19.6635 0.525318 19.9625 0.991984 19.9625H2.72532C12.8587 19.9625 17.4587 14.9459 17.5587 3.71667L19.6587 1.72332C20.0587 1.3911 20.092 0.793093 19.7587 0.394423Z"
```

Comment:
- kindly, use a ul tag to group the list of social icons, and li per icon; each icon has a link (a tag)for sematic reasons 👌